r-interface/reticulate/reticulate.md

```diff
@@ -0,0 +1,163 @@
+Using up-to-date Gudhi library in R
```

Comment:
Is it possible with quarto, when generating `*.md` files, to force the file name to REDME.md, instead of reticulate.md ?
Thanks to this modification, one can see the md file rendered when going into the [reticulate directory](https://github.com/astamm/TDA-tutorial/tree/r-interface/r-interface/reticulate)